cucumber/authenticators_jwt/features/authn_jwt_validate_restrictions.feature

```diff
@@ -1,4 +1,8 @@
 Feature: JWT Authenticator - Validate restrictions
+
+  Tests to check that host annotations are validated correctly in jwt authenticator. Focusing on checking that
+  the annotations that are relevant to the vendor are the only being checked.
```

Comment:
Focusing on checking that only the vendor related annotations are being checked.